.eslintignore

```diff
@@ -6,6 +6,7 @@ coverage
 .DS_Store
 .code
 *.iml
+package.json
```

Comment:
Why ignore this?

Reply:
I just noticed that eslint was still trying to lint it without any `json` plugin (or at least with the current repo configs). It was mainly to prevent vscode to spam an un-existing error.

Also, I still need to read `v9`'s changelog, so I'm not even sure if this is still a thing.